pkg/util/util.go

```diff
@@ -41,3 +42,20 @@ func GenerateRandomString(n int) string {
 	}
 	return string(b)
 }
+
+// Hyphenate applicationName and componentName
+func NamespaceComponent(componentName string, applicationName string) (string, error) {
```

Comment:
I have an issue with how this function is named. As I see this. It is not namespacing a component.
It is generating name that will be used for OpenShift objects that are representing component.